perceval/backends/jenkins.py

```diff
@@ -48,15 +48,17 @@ class Jenkins(Backend):
     :param url: Jenkins url
     :param tag: label used to mark the data
     :param cache: cache object to store raw data
+    :param blacklist_jobs: exclude the jobs of this list while fetching
     """
-    version = '0.3.0'
+    version = '0.3.1'
```

Comment:
The version should be `0.4.0` because you're adding a new feature and not fixing bugs.